src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -83,6 +83,13 @@ function updateBundleDepositsV3(dict: BundleDepositsV3, deposit: V3DepositWithBl
   if (!dict?.[originChainId]?.[inputToken]) {
     assign(dict, [originChainId, inputToken], []);
   }
+  if (
+    dict[originChainId][inputToken].some(
+      (d) => d.transactionHash === deposit.transactionHash && d.logIndex === deposit.logIndex
+    )
+  ) {
+    throw new Error("Duplicate deposit in bundleDeposits");
```

Comment:
Can we add a log message here to identify the relevant deposits? That will speed up debugging time significantly if we're able to isolate them.

Reply:
bf3bae61529cd5f2f1adda0e398199eabc09a75d